beetsplug/convert.py

```diff
@@ -90,7 +90,7 @@ def get_format(fmt=None):
     return (command.encode('utf-8'), extension.encode('utf-8'))
 
 
-def should_transcode(item, fmt):
+def should_transcode(self, item, fmt):
```

Comment:
Added `self` to allow logging calls. Not 100% sure of the ramifications of this.

Reply:
FWIW, I think it's a little more common to just pass around the log rather than the entire `self` object. (And if we were going to do that, I might name it something like `plugin` instead of `self`, which could be misinterpreted.)